calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -178,39 +178,36 @@ Now if a SQL Injection attack is detected for rule ID 942100, you will see the g
 
 #### Change WAF rules
 
-By default, the WAF OWASP Core Rule set is configured to log a warning if there is an attack. However, if you want traffic for the service to also be denied or dropped, 
+By default, the WAF OWASP Core Rule set is configured to log all matched rules and block offending requestis using Anomaly Scoring Mode. However, if you want traffic for the service to not be blocked and only logged,
 you must edit the modsecurity rules. 
 
-#### Deny traffic
+#### Allow traffic
 
-For example, if you want to deny traffic with a 403 response, do the following steps: 
+For example, if you want to allow traffic, do the following steps:
 
-1. Modify the modsecurity core rulesets' `SecDefaultAction` to deny rules by commenting and uncommenting these sections in the `crs-setup.conf`, 
-which will change the `SecDefaultAction` from `pass` to `deny`.
+1. Change the disruptive action, on file `RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example` section `'Changing the Disruptive Action for Anomaly Mode'`.
```

Comment:
This step is unclear.

Reply:
I think this should be 'Anomaly scores could result in Disruptive action. To change the disruption action, see the file `RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example` section `'Changing the Disruptive Action for Anomaly Mode'`'